SushiTime/Assets/SystemAssets/ScreenSystem/Scripts/Runtime/CallHomeScreen.cs

```diff
@@ -0,0 +1,20 @@
+namespace ScreenSystem
+{
+    using System;
+    using System.Collections;
+    using System.Collections.Generic;
+    using UnityEngine;
+
+    /// <summary>
+    /// Call the home screen. May instant close
+    /// all other screens, or may invokve a modal
+    /// screen first.
+    /// </summary>
+    public class CallHomeScreen : MonoBehaviour
+    {
+        public void OnCallHomeScreen(bool modalMode)
```

Comment:
What is Modal Mode? You didn't specify what that is. Not everybody is a UX designer, and they may not know "modal"